desktop-widgets/tab-widgets/TabDiveInformation.h

```diff
@@ -5,6 +5,8 @@
 #include "TabBase.h"
 #include "core/subsurface-qt/DiveListNotifier.h"
 
+#define INFOBLUE #0000c0
+
```

Comment:
again, the un-named `#0000c0` - would it make more sense to just use `medium blue`?

Reply:
Done.